postgres.go

```diff
@@ -358,87 +359,100 @@ func (store *PostgresStore) SaveCountryStats(tx *sqlx.Tx, entity *CountryStats)
 }
 
 // Session implements the Store interface.
-func (store *PostgresStore) Session(fingerprint string, maxAge time.Time) time.Time {
-	query := `SELECT "session" FROM "hit" WHERE fingerprint = $1 AND "time" > $2 LIMIT 1`
+func (store *PostgresStore) Session(params QueryParams, fingerprint string, maxAge time.Time) time.Time {
+	params.validate()
+	query := `SELECT "session"
+		FROM "hit"
+		WHERE ($1::bigint IS NULL OR tenant_id = $1)
+		AND fingerprint = $3
+		AND "time" AT TIME ZONE $2 > $4 LIMIT 1`
```

Comment:
Wouldn't it be less error prone to simply store the time either in a fixed timezone or use `time(stamp) with time zone` as type?